README.md

```diff
@@ -71,6 +71,22 @@ At this point, your local branch is synced to the original repository's master b
 ```
 git push origin master
 ```
+
+#### Alternative
+
+Set up  the original repository as your upstream.
+```
+git remote add upstream https://github.com/The-Codesis/simple-static-website.git
+```
+Now, fetch and merge the changes from the original repository simultaneously.
+```
+git pull upstream master
```

Comment:
@pranshi112 can you look for the difference between `git pull upstream` and `git fetch upstream`

Reply:
What kind of changes? Please elaborate.

Reply:
git fetch upstream do not merge it with your local repo, just download the changes whereas git pull upstream downloads and merges them in one go, you can add that if you want

Reply:
@tushar5526 Right

Reply:
@tushar5526 Right